### DISCUSSION TITLE
Unable to find documentation to use @DocumentID

### DISCUSSION THREAD

**Discussion by nathvarun** [firebase/firebase-ios-sdk#10563] [Q&A]
I can't seem to find any documentation on how to use the library.  
For eg. For firebasefirestoreswift I see @DocumentID being used but there is no documentation regarding that. 
Maybe I'm missing it somewhere.

Would appreciate if someone can point me in the right direction

**Comment by andrewheard** [comment]
Hi @nathvarun, sorry the documentation is mixed in with the implementation which makes it harder to find. Please take a look at [DocumentID.swift](https://github.com/firebase/firebase-ios-sdk/blob/67ae9555da3f5ff9f432485c1fc09e2dcd0b58da/Firestore/Swift/Source/Codable/DocumentID.swift#L69-L104), which has more details. And let us know if it's still unclear. 🙂
